gitlab/v4/objects/repositories.py

```diff
@@ -233,6 +235,8 @@ def repository_archive(
         query_data = {}
         if sha:
             query_data["sha"] = sha
+        if path is not None:
```

Comment:
Will need to change the name of the other `path` variable on line 232 and on line 241.

Reply:
Maybe to `url_path`